recipes/orange3-imageanalytics/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "Orange3-ImageAnalytics" %}
+{% set version = "0.1.13" %}
+{% set sha256 = "86a6bc65617b6796af764846d55e0c1bc49960716e673970a62beea3b6f60e10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Think this one can be `noarch:python`.

Reply:
Orange can access documentation online so it is fine.

Reply:
It can't be noarch: python since skip parameter is present (this addon is python3 only)